xc/device.ml

```diff
@@ -1483,6 +1483,30 @@ module Dm = struct
     extras: (string * string option) list;
   }
 
+  module Profile = struct
+    type t = Qemu_trad | Qemu_upstream_compat | Qemu_upstream
+    let fallback = Qemu_trad
+    module Name = struct
+      let qemu_trad            = "qemu-trad"
+      let qemu_upstream_compat = "qemu-upstream-compat"
+      let qemu_upstream        = "qemu-upstream"
+      let all = [ qemu_trad; qemu_upstream_compat; qemu_upstream ]
+    end
+    let wrapper_of = function
+      | Qemu_trad            -> "/usr/libexec/xenopsd/qemu-dm-wrapper"
```

Comment:
Please use the path from `Resources`. For this one, we can keep the `qemu_dm_wrapper` that was already there. The two below can be added.

Reply:
@robhoes Added the Resources change to f38664c008094152bd0b39f8401c8c51017c8c93